src/pymor/vectorarrays/block.py

```diff
@@ -178,8 +189,9 @@ def __eq__(self, other):
     def dim(self):
         return sum(subspace.dim for subspace in self.subspaces)
 
-    def zeros(self, count=1, reserve=0):
-        return BlockVectorArray([subspace.zeros(count=count, reserve=reserve) for subspace in self.subspaces], self)
+    def zeros(self, count=1, reserve=0, dtype=None):
+        return BlockVectorArray([subspace.zeros(count=count, reserve=reserve, dtype=None)
+                                 for subspace in self.subspaces], self)
```

Comment:
I would assume that for mixed precision algorithms you would want to have good control where which precision is used, so the precision should probably be part of the `VectorSpace`. - Since copies are usually relatively cheap, I would agree to remove the `dtype` argument until we see a specific need for the functionality.